SingularityService/src/main/java/com/hubspot/singularity/resources/StateResource.java

```diff
@@ -56,4 +59,11 @@ public SingularityState getState(@QueryParam("skipCache") boolean skipCache, @Qu
   public Optional<SingularityTaskReconciliationStatistics> getTaskReconciliationStatistics() {
     return stateManager.getTaskReconciliationStatistics();
   }
+
+  @GET
+  @Path("/ui-configuration")
+  @ApiOperation("Retrieve information about the deployed UI configuration")
+  public UIConfiguration getUIConfiguration() {
```

Comment:
When we create the index view, there is a slightly more hydrated version of ui configuration that gets sent. We would probably want a new pojo that mimics that. It includes things like shell commands/etc from different ares of singularity configuration